spec/helpers/measures_helper_spec.rb

```diff
@@ -153,6 +153,12 @@
         it { is_expected.to match 'The price of your goods does not exceed' }
       end
 
+      context 'with eps condition code V' do
+        let(:condition) { build :measure_condition, :eps, condition_code: 'V' }
```

Comment:
```suggestion
        let(:condition) { build :measure_condition, :eps }
```

We should move the condition code V to the factory